garden/migration_from_ignition.md

```diff
@@ -461,6 +461,26 @@ gz sim shapes.sdf
 
 Notice that the `gazebo` verb is **deprecated**.
 
+#### Helpful CLI Redirection
+
+In order to support side-by-side installs of `Garden` and `Fortress`, the `ign` CLI executable will not be installed for `Garden`.
+This means that unless `Fortress` is also installed alongside `Garden`, CLI usage has to be migrated.
+
+You may add the following script to your `~/.bashrc` file to redirect any `ign` calls to `gz`, so you won't have to migrate all of your scripts (that are still using `ign`), though it is still recommended to make the migration.
+
+```shell
+ign() {
+  if which ign &> /dev/null; then
+    if which gz &> /dev/null; then
+      echo "[DEPRECATED] ign is deprecated! Please use gz instead!"
```

Comment:
I think the deprecation warning should be below. If the user lands here, they have `ign` installed, which means they have Citadel / Fortress, and using `ign` in this case isn't deprecated.

The deprecated situation is below, when they don't have `ign`, but have `gz`, and still want to use `ign`. Right?

Reply:
I should've noted that this change was already made, even though GitHub isn't tracking it...

https://github.com/gazebosim/docs/pull/289/commits/1551f003e98981d53a9f5f7c62e2223ce0530881#diff-fd86d9016d73bca6146828475688b7d155b1eb824b360635b54950b2a161971fR478